File: src/com/Torvald/Terrarum/ConsoleCommand/QuitApp.java
package com.Torvald.Terrarum.ConsoleCommand;

/**
 * Created by minjaesong on 16-01-15.
 */
public class QuitApp implements ConsoleCommand {

    @Override
    public void execute(String[] args) {
        System.exit(1);
    }

    @Override
    public void printUsage() {

    }
}

File: src/net/torvald/terrarum/console/QuitApp.kt
package net.torvald.terrarum.console

import com.badlogic.gdx.Gdx

/**
 * Created by minjaesong on 2016-01-15.
 */
internal object QuitApp : ConsoleCommand {

    override fun execute(args: Array<String>) {
        Gdx.app.exit()
    }

    override fun printUsage() {

    }
}
